godtossing/Pos/Binary/GodTossing/Core.hs

```diff
@@ -20,16 +20,14 @@ import           Pos.Ssc.GodTossing.Core.Types (Commitment (..), CommitmentsMap
 import           Serokell.Util                 (allDistinct)
 
 instance Bi Commitment where
-  encode Commitment{..} = encodeListLen 3 <> encode commShares
```

Comment:
Was it a bug? Does it work in master? Do I understand correctly that it produces invalid CBOR but works by chance?

Reply:
Ok, I read diff incorrectly.